bootstrap.yml

```diff
@@ -1133,6 +1135,28 @@ packages:
           'DESTDIR': '@THIS_COLLECT_DIR@'
         quiet: true
 
+  - name: cxxshim
```

Comment:
Missing metadata, please add it. We should add missing metadata whenever we’re updating a package. I guess the maintainer of both frigg and cxxshim would be @avdgrinten?